provider/maas/volumes.go

```diff
@@ -34,6 +33,19 @@ const (
 	tagsAttribute = "tags"
 )
 
+// StorageProviderTypes implements storage.ProviderRegistry.
+func (*maasEnviron) StorageProviderTypes() []storage.ProviderType {
+	return []storage.ProviderType{maasStorageProviderType}
+}
+
+// StorageProvider implements storage.ProviderRegistry.
+func (*maasEnviron) StorageProvider(t storage.ProviderType) (storage.Provider, error) {
+	if t == maasStorageProviderType {
+		return maasStorageProvider{}, nil
+	}
+	return nil, errors.NotFoundf("storage provider %s", t)
```

Comment:
haven't been mentioning it everywhere, but following up from the note on the error test way back up above, I think these should all be %q